config/config.go

```diff
@@ -398,6 +391,21 @@ func getDefaultStringArray(envvar string, defaultValue string) (retVal []string)
 	return
 }
 
+func getDefaultStringMap(envvar string, defaultValue string) (retVal map[string]string) {
+	csv := os.Getenv(envvar)
+	if csv == "" {
+		csv = defaultValue
+	}
+	retVal = map[string]string{}
+	for _, token := range strings.Split(csv, ",") {
+		mapEntry := strings.Split(token, "=")
+		if len(mapEntry) == 2 {
```

Comment:
you should add an else-stmt here and in that else you should log a warning message here because if the length here is 1, it means the person misconfigured this (forgot the "=" for example -- "aa=bb,cc,dd" instead of "aa=bb,cc=dd")

Reply:
Added warning.